lib/server.js

```diff
@@ -50,6 +52,9 @@ module.exports = (config, callback) => {
         cache: config.cache
     });
 
+    server.app = {
+        ecosystem: config.ecosystem
```

Comment:
Looks like `config.ecosystem` is missing from JSDocs comment. Can you help add it?

Reply:
I forgot to remove these lines. I added config.ecosystem for requesting to the API. But I removed a function that request to the API, these lines are not needed. I removed those.